src/main/java/org/example/c17inputoutput/ReadingAFileChallenge.java

```diff
@@ -0,0 +1,85 @@
+package org.example.c17inputoutput;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class ReadingAFileChallenge {
+    public static void main(String[] args) {
+
+        try (BufferedReader br = new BufferedReader(
+                new FileReader("testfile.txt"))) {
+
+//            System.out.printf(" %,d lines in file%n", br.lines().count());
+
+            Pattern pattern = Pattern.compile("\\p{javaWhitespace}+");
```

Comment:
Give this Pattern object a suggestive name.

Reply:
done. New name is splitToWords

Reply:
Not good. Name should describe the pattern, not how the pattern is used. This pattern can be used for many things not only for splitting words. You can call this pattern is for `oneOrManyWhitespaces`.

Reply:
ok. change it